View/register_page.php

```diff
@@ -0,0 +1,32 @@
+<!-- HTML part of register page -->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap');
+  </style>
```

Comment:
As sir discussed in the meeting everything should be separated (HTML, CSS) don't write inner CSS try to link the external CSS.